src/hb-ot-color-sbix-table.hh

```diff
@@ -334,7 +336,8 @@ struct sbix
   bool add_strike (hb_subset_context_t *c,
                    const void *dst_base,
                    LOffsetTo<SBIXStrike>* o,
```

Comment:
Just notice that the only thing that happens with the 'o' parameter in this call is that it's set to 0. Recommend that you remove it, and do the set to zero in the serialize_strike_offsets method instead.

Reply:
Done.

Reply:
Done.